presto/presto_test.go

```diff
@@ -275,6 +275,52 @@ func TestQueryFailure(t *testing.T) {
 	}
 }
 
+func TestExec(t *testing.T) {
+	c := &Config{
+		PrestoURI:         "http://foobar@localhost:8080",
+		SessionProperties: map[string]string{"query_priority": "1"},
+	}
+	dsn, err := c.FormatDSN()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	db, err := sql.Open("presto", dsn)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer db.Close()
+
+	rows, err := db.Query(`SELECT * FROM "presto.execution:name=queryexecution"`)
```

Comment:
If we have access to `tpch` connector in this test I would prefer to use that instead `jmx`

Reply:
Done!